README.md

```diff
@@ -42,15 +42,15 @@ it should be the only directory to know about the store's implementation.
 
 ### `ui/`
 
-This is for presentational components.
-It should not import any utilities from `views/` or `store/`.
-It should also not know about the routes.
+This is for presentational building block components.
+Some components in the `ui/` directory should potentially connect to the store or routes,
+but in general they should be reusable and relatively isolated from the state.
 
 ### `views/`
 
-This is for connected and routed components.
+This is for routed and connected components.
 It can import from `ui/` and `store/` as needed.
-Components in this directory will likely know about the current routing.
+Components in this directory will likely be tied to routes, such as pages.
```

Comment:
I didn't realize the ESLint settings enforced the structure, so the original message was correct.